maven clean install fails due to file encoding

The maven-compiler plugin assumes UTF-8 encoded files by default, and the source files of this project are in iso-8859-1 (also known as latin1).  This can be resolved with additional configuration in the pom.xml.

`[INFO] --- maven-compiler-plugin:2.3.2:compile (default-compile) @ smm-tester ---
[INFO] Compiling 339 source files to /home/chris/projects/aira/smm-tester/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/chris/projects/aira/smm-tester/src/main/java/org/immregistries/smm/tester/manager/tximmtrac/ImmTracMessage.java:[1,13] error: unmappable character (0xA9) for encoding UTF-8
[ERROR] /home/chris/projects/aira/smm-tester/src/main/java/gov/nist/healthcare/hl7ws/messagevalidation/MessageValidationV2Interface.java:[177,51] error: unmappable character (0x93) for encoding UTF-8
[ERROR] /home/chris/projects/aira/smm-tester/src/main/java/gov/nist/healthcare/hl7ws/messagevalidation/MessageValidationV2Interface.java:[177,57] error: unmappable character (0x94) for encoding UTF-8
[ERROR] /home/chris/projects/aira/smm-tester/src/main/java/org/immregistries/smm/tester/manager/tximmtrac/ImmTracSegment.java:[1,13] error: unmappable character (0xA9) for encoding UTF-8
[INFO] 4 errors 
`

--- a/pom.xml
+++ b/pom.xml
@@ -203,6 +203,7 @@
 					<target>1.7</target>
 					<optimize>true</optimize>
 					<debug>true</debug>
+                    <encoding>iso-8859-1</encoding>
 				</configuration>
 			</plugin>
 			<plugin>

